test/functional/Java21andUp/src/org/openj9/test/jep425/VirtualThreadTests.java

```diff
@@ -51,6 +51,16 @@ public class VirtualThreadTests {
 
 	public static native boolean lockSupportPark();
 
+	private void incrementalWait(Thread t) throws InterruptedException {
+		/* Incrementally wait for 10000 ms. */
+		for (int i = 0; i < 20; i++) {
```

Comment:
I like the idea. On a reasonable machine is a 500ms wait usually necessary? I'm thinking the wait time could be decreased and the number of iterations increased so the test could normally take less time to complete.

Reply:
Yes, smaller increments will be better. Reduced the wait time by a factor of 10 and accordingly increased the number of iterations.